src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlClientInstrumentationEventSource.cs

```diff
@@ -75,5 +75,26 @@ public void EnrichmentException(string exception)
         {
             this.WriteEvent(5, exception);
         }
+
+        [Event(6, Message = "Command is filtered out.", Level = EventLevel.Verbose)]
+        public void CommandIsFilteredOut(string eventName)
```

Comment:
eventName is not used in the Message?
Also, its activity name.

Reply:
amended...